wagtailautocomplete/edit_handlers.py

```diff
@@ -16,23 +21,42 @@ def _can_create(page_type):
     ))
 
 
-class AutocompletePanel:
-    def __init__(self, field_name, page_type='wagtailcore.Page', is_single=True):
-        # is_single defaults to True in order to have easy drop-in
-        # compatibility with wagtailadmin.edit_handlers.PageChooserPanel.
-        self.field_name = field_name
-        self.page_type = page_type
-        self.is_single = is_single
-
-    def bind_to_model(self, model):
-        can_create = _can_create(self.page_type)
-        base = dict(
-            model=model,
-            field_name=self.field_name,
-            widget=type(
+if 'BaseFieldPanel' in locals():
```

Comment:
Is there a more robust or canonical way to perform this check? Does the wagtail package export a version constant?

Reply:
oh jk you figured this out in the next commit